src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/PeriodSchemes/DefaultPeriodSchemeCacheItem.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace EasyAbp.BookingService.PeriodSchemes;
+
+[Serializable]
+public class DefaultPeriodSchemeCacheItem
+{
+    public const string Key = "DefaultPeriodScheme";
+
+    public PeriodScheme Value { get; set; }
```

Comment:
Don't use an entity object as the cache item.

https://github.com/abpframework/abp/search?q=cacheitem